commit 0797a6f8754db982b87a1df63975ccf76df2905f
Author: Leland Lucius <lllucius@users.noreply.github.com>
Date:   Mon Jul 6 23:54:22 2015 +0200

    Fix accessibility support in wxMSW.
    
    Return the correct result from handing WM_GETOBJECT message.
    
    Closes #17058.
    
    (this is a backport of 6a435345f53732ed2a40dccc302e7d1f137e5eaa from master)

diff --git a/src/msw/window.cpp b/src/msw/window.cpp
index 7943f62..a141051 100644
--- a/src/msw/window.cpp
+++ b/src/msw/window.cpp
@@ -3386,7 +3386,8 @@ wxWindowMSW::MSWHandleMessage(WXLRESULT *result,
 
                 if (dwObjId == (LPARAM)OBJID_CLIENT && GetOrCreateAccessible())
                 {
-                    return LresultFromObject(IID_IAccessible, wParam, (IUnknown*) GetAccessible()->GetIAccessible());
+                    processed = true;
+                    rc.result = LresultFromObject(IID_IAccessible, wParam, (IUnknown*) GetAccessible()->GetIAccessible());
                 }
                 break;
             }
